### DISCUSSION TITLE
Leitor Cód de Barras

### DISCUSSION THREAD

**Discussion by diegorcampos** [RamonSilva20/mapos#1875] [Ideas]
Bom dia.
É possível inserir ou adaptar leitor de código de barras no app ?
Se sim, poderia me guiar de como fazer essa inserção ?

**Comment by jsfachini** [comment]
Boa tarde, No menu produtos tem a opção de gerar etiquetas

**Reply by diegorcampos** [reply]
então, tem pra gerar, mas aceita o leitor ?
digo, aceita o hardware ?
ou terei q alterar algo no codigo para q o sistema reconheça e aceite ?
